ops.py

```diff
@@ -9,20 +9,22 @@ class GGMLTensor(torch.Tensor):
     """
     Main tensor-like class for storing quantized weights
     """
-    def __init__(self, *args, tensor_type, tensor_shape, patches=[], **kwargs):
+    def __init__(self, *args, tensor_type, tensor_shape, patches=[], ggml_tensor_name=None, **kwargs):

```

Comment:
Why are we storing tensor_name as an attribute? This should be handled by the state dict and there's no reason it should ever be part of the tensor-like object.

Reply:
i can remove it if you want. my thinking was it could make debugging a lot easier since currently if you have issues with a `GGMLTensor` there's really no way to tell where it came from.

Reply:
Fair enough, think you should remove it in the final version, it's just another thing to be copied around and if by chance the tensor attributes get stripped then this one will just be `None` as well.